randovania/game_connection/connector_builder_choice.py

```diff
@@ -31,8 +30,6 @@ def is_usable(self) -> bool:
             # see: https://github.com/python/mypy/issues/8166
             if sys.platform == "darwin":
                 return False
-            if sys.platform == "linux" and randovania.is_frozen():
-                return os.getuid() == 0 and not randovania.is_flatpak()
```

Comment:
I don't have any way of testing the flatpak version here, so I'm not 100% confident in this. I *suspect* that if dolphin is installed in a way that lets it be launched from in the flatpak, this will work, but I don't know (a) if that's possible or (b) how to do it, although passing comments on other flatpak-based software imply that it might be possible.

Reply:
This won't work. Unless you make your own fork of randovania-flatpak that also includes dolphin-flatpak this doesn't work because the only way for flatpak applications to launch other applications is by signalling it to the host system. At which point it does not become a child process anymore.

There is a small discussion I once had with some gnome terminal devs in #flatpak:matrix.org, from what I recall their solution was to essentially have a daemon on the host system.